pkg/admissioncontroller/webhooks/admission/resourcesize/admission_test.go

```diff
@@ -60,9 +60,9 @@ var _ = Describe("handler", func() {
 		projectsSizeLimit, _ = resource.ParseQuantity("0M")
 		secretSizeLimit, _   = resource.ParseQuantity("1Mi")
 		// size of shoot w/ namespace, name, w/o spec
-		shootsv1beta1SizeLimit, _ = resource.ParseQuantity("405")
+		shootsv1beta1SizeLimit, _ = resource.ParseQuantity("429")
```

Comment:
Well, the test "should pass because size is in range for v1beta1 shoot" checks the size of a more or less empty shoot specification. However, with this change `,"systemComponents":null` is added. Therefore, the resourceSize is 24 bytes bigger.
(I also increased the v1alpha1 size for consistency. No test actually checks it.)

Reply:
:-) Thanks!